src/events/crawler/scrapers/AUS/index.js

```diff
@@ -5,8 +5,21 @@ import * as transform from '../../lib/transform.js';
 // Set county to this if you only have state data, but this isn't the entire state
 // const UNASSIGNED = '(unassigned)';
 
+const idMap = {
+  'Australian Capital Territory': 'id3:AU-ACT',
+  'New South Wales': 'id3:AU-NSW',
+  'Northern Territory': 'id3:AU-NT',
+  Queensland: 'id3:AU-QLD',
+  'South Australia': 'id3:AU-SA',
+  Tasmania: 'id3:AU-TAS',
+  Victoria: 'id3:AU-VIC',
+  'Western Australia': 'id3:AU-WA'
+};
+
 const scraper = {
-  country: 'AUS',
+  country: 'id0:AU',
+  scraperId: 'id0:AU',
+  id: 'id0:AU',
```

Comment:
How does one know/set these ids and what is the difference between these three?

Reply:
1. Getting the ids is easy, a contributor just looks up the right value on https://github.com/hyperknot/country-levels/blob/master/docs/country_list.md
2. Currently there is no different, it's a workaround to make the current system running as well as to use the new id system. It's quite verbose and not elegant though.